packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -31,7 +32,14 @@ export 'package:flutter/rendering.dart' show SelectionChangedCause;
 /// (including the cursor location).
 typedef SelectionChangedCallback = void Function(TextSelection selection, SelectionChangedCause cause);
 
-const Duration _kCursorBlinkHalfPeriod = Duration(milliseconds: 500);
+// The time it takes for the cursor to fade from fully opaque to fully
+// transparent and vice versa. A full cursor blink, from transparent to opaque
+// to transparent, is twice this duration.
+const Duration _kCursorFadeInAndOutTime = Duration(milliseconds: 500);
```

Comment:
We're using this even if we don't "fade" strictly right? I don't have anything better to suggest, let's just leave this name as is.

Reply:
yah, should I change the name back?

Reply:
ya, go ahead. Sorry